src/components/Project/Releases/index.js

```diff
@@ -98,12 +97,18 @@ class ReleaseView extends React.Component {
     if (seconds < 10) {
       seconds = "0" + seconds
     }
-
-    if(parseInt(hours, 10) < 1) {
-      return minutes+':'+seconds;
-    } else {
-      return hours+':'+minutes+':'+seconds;
+    
+    if(parseInt(hours,10) < 1) {
+      if(parseInt(minutes, 10) < 1) {
+        if(parseInt(seconds, 10) < 1){
+          return "No information on runtime."
+        }
+        return seconds + ' sec';  
+      }
+      return minutes+' min. '+seconds + ' sec.';
     }
+    
+    return hours+' h. '+minutes+' min. '+seconds + ' sec.';
```

Comment:
Can probably abbreviate just 'h' 'm' 's' for sake of space. Do you need to parse seconds? Isn't it available just above?

Reply:
Yes. You do integer math on it, then prepend a 0 (and convert to string), then parse it back to an int, then convert it back to a string. 

In fact you do that with hours and minutes field as well now that I look more closely at it.

You can keep everything as integers and then return a string value and at that same time pad the 0s with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart